### ISSUE TITLE
Bibliography style and links

### ISSUE THREAD

**Issue by larsvilhuber** [amichuda/crress-book#9]
Can you look into Chicago style bibliography that expands first names? 

Also, the citations themselves in the text (e.g., "As per Author (2022), blablabla..." ) are not hyperlinked. That seems oddly old-fashioned. Does Quarto not do that?

**Comment by amichuda** [comment]
The citations have been an issue from the begining. There's no integrated
way to extract bibliography from word and then add it in with pandoc. The
only thing I can think of now is to do it by regex, but I haven't had the
time.

**Comment by larsvilhuber** [comment]
Weren't you using Bib files? And doesn't quarto handle bib files?

--
Lars Vilhuber, Economist
Cornell University
p: +1.607-330-5743
https://calendly.com/larsvilhuber
My working day may not be your working day. Please respond during your working day.
________________________________

**Comment by larsvilhuber** [comment]
https://quarto.org/docs/authoring/footnotes-and-citations.html

And yes, you or somebody else needs to build the bib file from the Word document. That's just how this works.

--
Lars Vilhuber, Economist
Cornell University
p: +1.607-330-5743
https://calendly.com/larsvilhuber
My working day may not be your working day. Please respond during your working day.
________________________________

**Comment by amichuda** [comment]
Bib files don't fix the whole issue because you need the citation key in
the markdown for it to register the citation.

**Comment by larsvilhuber** [comment]
Yup. By Hand. I've done for it a different book with 20 chapters. That's why we prefer LaTeX...

Bib can be generated here: https://update.lib.berkeley.edu/2018/02/07/extracting-references-from-an-already-created-bibliography/

**Comment by larsvilhuber** [comment]
I'm going to suggest we shelve this. If we are submitting these to HDSR, they'll have additional stuff, no need to do that here right now.
